scripts/browserstack_tests/browserstack_frontend_test1.rb

```diff
@@ -0,0 +1,39 @@
+require 'rubygems'
+require 'selenium-webdriver'
+
+# Input capabilities
+caps = Selenium::WebDriver::Remote::Capabilities.new
+#caps["browser"] = "IE"
```

Comment:
Let's try updating this to IE11, the whole purpose for this is to test IE not Chrome so that's important to update.